.travis.yml

```diff
@@ -22,12 +21,15 @@ script:
   - 'if [ "${JOB}" = "checkapidocs" ]; then make check-api-doc; fi'
   - 'if [ "${JOB}" = "validateapispec" ]; then make validate-api-spec; fi'
   - 'if [ "${JOB}" = "validatecodegen" ]; then ./hack/verify-codegen.sh; fi'
+  - 'if [ "${JOB}" = "verifyvendor" ]; then ./hack/verify-vendor.sh; fi'
 env:
   global:
+    - GO111MODULE=on
     - GOARCH=amd64
-    - GO_FOR_RELEASE=1.11
+    - GO_FOR_RELEASE=1.12
     - IMAGE_NAME=tsuru/api
```

Comment:
add `- GOPROXY=https://proxy.golang.org` and get faster builds :grin:

Reply:
Wow, it's a LOT faster. Thanks!

Reply:
You can probably delete the vendor folder. Trust the cloud.

https://www.youtube.com/watch?v=AnxrJiS5uKU